sonaranalyzer-dotnet/build/rspec-templates/CSharpTestTemplate.cs

```diff
@@ -30,7 +30,8 @@ public class $DiagnosticClassName$Test
         [TestCategory("Rule")]
         public void $DiagnosticClassName$()
         {
-            Verifier.VerifyAnalyzer(@"TestCases\$DiagnosticClassName$.cs", new $DiagnosticClassName$());
+            Verifier.VerifyAnalyzer(@"TestCases\$DiagnosticClassName$.cs",
```

Comment:
It is just that lately I often ended-up with long names so I had to put them on the line.